javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,80 @@
+//Alireza hassanloo
+//Age-ify (A future age calculator)
+
+const yearOfBirth = 1979;
+const yearFuture = 2037;
+const age = yearFuture - yearOfBirth;
+
+//first way
+console.log(`I will be ${age} years old in ${yearFuture}`);
+//second way
+console.log("I will be", age, "years old in", yearFuture);
```

Comment:
If you actually see this in the console, it would look exactly like this: `"I will be" 58 "years old in" 2037`
The correct way to to string literals is with the plus sign
```suggestion
console.log("I will be " + age + " years old in " + yearFuture);
```

The result then would look like this: `I will be 58 years old in 2037`